codecov.yml

```diff
@@ -1,7 +1,12 @@
 coverage:
   status:
     project:
+      # no leniency on `main`
+      main:
+        branches:
+          - main
+          - gh-readonly-queue/main/*
+        target: 100%
+      # on PRs, we allow some leniency since integration tests haven't run yet
       default:
-        target: auto
-      minimum:
-        target: "100%"
+        target: 98%
```

Comment:
@ericphanson This can be 99, right, not 98?

Reply:
I dunno, I thought 98 was safer. We can try 99 though, if this merges down then it seems ok

Reply:
```suggestion
        target: 99%
```